presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -30,3 +30,8 @@ The following can be helpful for diagnosing garbage collection (GC) issues:
     -XX:+PrintAdaptiveSizePolicy
     -XX:+PrintSafepointStatistics
     -XX:PrintSafepointStatisticsCount=1
+
+If Java JMV Version > 9:
```

Comment:
Typo: JMV

Reply:
Also, it should be ">= 9". Maybe rephrase as: "If running on Java 9 or later:"

Reply:
@martint since `<9` is kind of passé, do we need this?